RELEASE.md

```diff
@@ -0,0 +1,95 @@
+# Release
+Follow the steps to release a new version of spyder-line-profiler.
+
+In the commands below, replace `X.Y.Z` with the release version when needed.
+
+**Note**: We use `pip` instead of `conda` here even on Conda installs, to ensure we always get the latest upstream versions of the build dependencies.
+
+## PyPI and GitHub
+
+You will need to have a local clone of the repo. The following steps supose a repo setup from a fork with and `upstream` remote pointing to the main `spyder-line-profiler` repo
+
+* Close [milestone on Github](https://github.com/spyder-ide/spyder-line-profiler/milestones)
+
+* Update local repo
+
+  ```bash
+  git restore . && git switch master && git pull upstream master
+  ```
+* Clean local repo
+
+  ```bash
+  git clean -xfdi
+  ```
+
+* Install/upgrade Loghub
+
+  ```bash
+  pip install --upgrade loghub
+  ```
+
+* Update `CHANGELOG.md` with Loghub: `loghub spyder-ide/spyder-line-profiler --milestone vX.Y.Z`
+
+* git add and git commit with "Update Changelog"
+
+* Update `__version__` in `__init__.py` (set release version, remove `dev0`)
+
+* Create release commit
+
+  ```bash
+  git commit -am "Release X.Y.Z"
+  ```
+
+* Update the packaging stack
+
+  ```bash
+  python -m pip install --upgrade pip
+  pip install --upgrade --upgrade-strategy eager build setuptools twine wheel
+  ```
+
+* Build source distribution and wheel
+
+  ```bash
+  python -bb -X dev -W error -m build
```

Comment:
I think this command won't work because we're not using `pyproject.toml` here. Did you check it?

Reply:
I just copy-pasted from qtpy 😅 although checking seems like the command works: 

![imagen](https://user-images.githubusercontent.com/16781833/169874139-e8058506-f671-4ec2-8308-0be4bee8594c.png)